README.md

```diff
@@ -4,25 +4,85 @@
+
+* *__Only A Single Command Line Interface Necessary__*
+
+  No need to spend time setting up complex development debugging, logging, and notification configurations. The terminal that runs your React Slingshot based project is all you'll need. Trim down on desktop clutter and tooling dependencies. The same CLI running your project doubles as an output console which continously streams relevant development data while you work.
+
+* *__No More "JavaScript Fatigue"__*
+
+  Driven by an active community of contributers, React Slingshot sees regular updates which include the latest, bleeding-edge development enhancements. The project itself is [built atop the most powerful, widely understood community resources available](#technologies) for working with React.
+
+* *__Example App With Scripts Included__*
+
+  Whether you're new to ReactJS and want to start learning, or feel like you don't quite yet have a strong grasp on working with ReactJS alongside Redux, or even if you're coming back to ReactJS/Redux after having spent time working in a completely different tech stack and just need a good refresher; React Slingshot has you covered. By default, any time you make a fresh clone of the React Slingshot Github repository, it comes pre-packaged with the codebase for an example app. If and when you decide you no longer need the example app anymore, and are ready to shave off excess code bloat, run built-in scripts that leave you with just the bare essentials needed for development. The example app itself isn't another basic "Todo List", or any other demo you're used to seeing. React Slingshot purposefully takes covering ReactJS/Redux development fundamentals a step further; Showcasing current best practices in using ReactJS alongside Redux, as well as common ReactJS design patterns like "containers", "smart" and "dumb" components, "contolled" versus "un-controlled" components, etc. No 3rd party libraries that abstract out any of the heavy lifting are used. You get a clear, "vanillaJS" approach thats demonstrates the ReactJS way of doing things. It also serves as mini-presentation for some of the functionality React Slingshot offers for rapid application development. In terms of what you can expect to learn, the example app provides minimal, yet effective, illustrations of a few complex ReactJS/Redux design concepts you'd commonly work to implement in many projects. These include:
```

Comment:
Not sure about the wording. Some of it sounds sales pitchy a bit. I'll let @coryhouse chime in on this.